tests/unit/AliasForm.spec.js

```diff
@@ -70,12 +70,13 @@ describe('AliasForm.vue', () => {
         expect($http.post).toBeCalledWith('/api/aliases/', {
           name: '',
         });
-        return wrapper.vm.nextTick();
+        return wrapper.vm.$nextTick();
       })
       .then(() => {
-        expect(wrapper.vm.aliases).toContain({
+        expect(wrapper.vm.aliases).toContainEqual({
           id: 3,
           name: '',
+          disabled: false,
```

Comment:
I really dislike negative sense properties. Reasoning about them tends to end up saddled with double-negatives. Can this be `enabled`?

Reply:
In this case, it reflects an HTML attribute, so I'm actually inclined to leave it this way, though generally I agree with you.